sync/src/lib.rs

```diff
@@ -47,9 +47,11 @@ impl Into<ProtocolId> for NetworkProtocol {
     }
 }
 
-//  Timeout = base + per_header * (expected number of headers)
-pub const HEADERS_DOWNLOAD_TIMEOUT_BASE: u64 = 6 * 60 * 1000; // 6 minutes
-pub const HEADERS_DOWNLOAD_TIMEOUT_PER_HEADER: u64 = 1; // 1ms/header
+pub const HEADERS_DOWNLOAD_TIMEOUT_BASE: u64 = 2 * 60 * 1000; // 2 minutes
```

Comment:
The name is borrowed from Bitcoin, however, I think it is misleading. It is not really a timeout to download headers, it is a timeout to sync the tip.

Reply:
I will rename this const variable.